toqito/matrix_ops/commutant.py

```diff
@@ -0,0 +1,109 @@
+"""Module for computing the commutant of a set of matrices."""
+
+import numpy as np
+from scipy.linalg import null_space
+
+
+def commutant(A: np.ndarray | list[np.ndarray]) -> list[np.ndarray]:
+    r"""Compute an orthonormal basis for the commutant algebra.
+
```

Comment:
Is there a source for the commutant definition?

Reply:
@fs1132429 Here's an example of how you can web links to the references. 

https://github.com/vprusso/toqito/blob/72849e98975de623353702fd3b80cd419b1cfca3/docs/refs.bib#L1413-L1417